pynicotine/__init__.py

```diff
@@ -90,6 +90,7 @@ def check_arguments():
 
     # Disables critical error dialog; used for integration tests
     parser.add_argument("--ci-mode", action="store_true", help=argparse.SUPPRESS)
+    parser.add_argument("--isolated-mode", action="store_true", help=argparse.SUPPRESS)
```

Comment:
"--isolated-mode" -> "--isolated"

A way of detecting this type of runtime environment automatically would be ideal, if possible?